### DISCUSSION TITLE
Basic questions about styling with Tailwind and Vue

### DISCUSSION THREAD

**Discussion by devopg** [Milkdown/milkdown#671] [Q&A]
Some basic question

1)  how override styles with tailwind ? 
if i add style in global  `@layer base`  and it dont work 

```
@layer base {
	
	
	.milkdown {
		height: 200px !important;
		max-height: 200px !important;
		/* overflow: scroll !important; */
	}
	
}
```

2) How apply styles with Vue3 with component with scoped? 

it dont work because after render div with milkdown receive prefix like  `class=milkdown-4ue7`
```
<template>
	<VueEditor :editor="editor" />
</template>

<style scoped>
.milkdown{
    max-height: 100px
}
</style>

```

**Comment by Saul-Mirone** [comment]
Maybe use `@apply` utility with `@layer components`?
https://tailwindcss.com/docs/reusing-styles#extracting-classes-with-apply

**Reply by devopg** [reply]
with `Layer components` dont work too
```
@layer components {
	.milkdown-menu-wrapper{
		position: relative;
	}
}
```

**Comment by z-950** [comment]
check [`Deep Selectors`](https://vuejs.org/api/sfc-css-features.html#scoped-css) for question 2.

**Reply by devopg** [reply]
It works, but it's not convenient (

**Reply by z-950** [reply]
It's the only way with scoped. I think you can try this. 
``` js
.wrapper :deep() {
  .milkdown {
    // ....
  }
}
```
It should be used with css preprocessors.
